test/distributed/test_c10d.py

```diff
@@ -1973,13 +1973,15 @@ def tearDown(self):
     def world_size(self):
         return 2
 
-    def _prepare_single_device_module(self, process_group, devices, device_ids, global_batch_size):
+    def _prepare_single_device_module(
```

Comment:
Shall we turn on more tests? like:

1. `test_multiple_outputs_multiple_backward`
2. `test_accumulate_gradients_no_sync`
3. `test_arbitrary_forward_return_value`
4. etc.

Will they work under this mode?